index.md

```diff
@@ -37,20 +37,20 @@ title: Sean Lo's Resume
 
 <span> San Francisco, CA  <span class="small-right">Nov. 2021 - Mar. 2024</span> </span>
 
-* Developed and maintained a Python Appium Client/Server pytest testing framework for PS4, PS5, Windows, and Web that was utilized by 1000+ of developers and testers
-* Implemented CI/CD release pipelines utilizing Jenkins, Bash, and Groovy, enabling automated semantic releases for 30+ teams delivering PlayStation React Native-based applications
-* Maintained a crucial PlayStation Network tool in JavaScript with Node.js and MongoDB, processing 100,000+ of calls weekly across API, web, and CLI for development and testing
-* Proactively led refactoring effort of multiple Python-based internal tools leveraged by 20+ teams daily for development, testing, and nightly regression jobs
+* Spearheaded the development and maintenance of a Python Appium Client/Server pytest testing framework for PS4, PS5, Windows, and Web that was utilized by 1000+ of developers and testers, leading to increased developer productivity and 20% less service downtime
+* Designed and implemented CI/CD a release pipeline utilizing Jenkins, Bash, and Groovy, enabling automated semantic releases for 30+ teams delivering PlayStation React Native-based applications
+* Supported a crucial PlayStation Network tool in JavaScript with Node.js and MongoDB, processing 100,000+ of calls weekly across API, web, and CLI for development and testing
+* Proactively led refactoring effort of multiple Python-based internal tools leveraged by 20+ teams daily for development, testing, and nightly regression jobs, resulting in a 44% decrease in job processing times and 70% increase in developer productivity
 * Crafted a Python-based Telemetry app with MongoDB and Grafana, enabling shareholder insights and improving KPI monitoring and decision-making
```

Comment:
what kind of insights? Any specific numbers or metrics on how it improved processes?